### ISSUE TITLE
Getting Error while using SES API keys in email config.

### ISSUE THREAD

**Issue by bkrajendra** [wintercms/wn-drivers-plugin#1]
I am trying to use SES in email config, Getting an Error while using SES API keys in email config.
```
"Class 'Aws\Ses\SesClient' not found" on line 98 of /myhosting.com/vendor/laravel/framework/src/Illuminate/Mail/TransportManager.php
```

**Comment by mjauvin** [comment]
@bkrajendra try adding this to wn-drivers-plugin/composer.json in "require:" section to solve the issue:
```
"aws/aws-sdk-php": "^3.155"
```

**Comment by mjauvin** [comment]
Try latest dev-main, that should fix your issue.

**Comment by mjauvin** [comment]
Just released version 2.0.1.

**Comment by bkrajendra** [comment]
Thank you! I updated to latest and its working fine now.

**Comment by bennothommo** [comment]
Fixed by #2.
